src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -116,6 +115,12 @@ function ReportActionItemMessageEdit(props) {
         isFocusedRef.current = isFocused;
     }, [isFocused]);
 
+    // We consider the report action in edit mode is active when
+    // - it is focused or
+    // - EmojiPicker's activeID is equal to this action's reportActionID or
+    // - ReportActionContextMenu's reportActionID is equal to this action's reportActionID
+    const isActive = useCallback((reportActionID) => isFocusedRef.current || EmojiPickerAction.isActive(reportActionID) || ReportActionContextMenu.isActiveReportAction(reportActionID), []);
```

Comment:
NAB. isActive does not need to accept a parameter, the report action id will always be `props.action.reportActionID`

Reply:
Updated